resources/i18n/it_IT/fdmextruder.def.json.po

```diff
@@ -124,74 +157,42 @@ msgctxt "machine_nozzle_id label"
-msgid "Nozzle X Offset"
-msgstr "Offset X ugello"
-
-msgctxt "machine_nozzle_offset_y label"
-msgid "Nozzle Y Offset"
-msgstr "Offset Y ugello"
-
-msgctxt "machine_extruder_start_code description"
-msgid "Start g-code to execute when switching to this extruder."
-msgstr "Inizio codice G da eseguire quando si passa a questo estrusore."
-
-msgctxt "extruder_prime_pos_x description"
-msgid "The X coordinate of the position where the nozzle primes at the start of printing."
-msgstr "La coordinata X della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_prime_pos_y description"
-msgid "The Y coordinate of the position where the nozzle primes at the start of printing."
-msgstr "La coordinata Y della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_prime_pos_z description"
-msgid "The Z coordinate of the position where the nozzle primes at the start of printing."
-msgstr "Indica la coordinata Z della posizione in cui l’ugello si innesca all’avvio della stampa."
-
-msgctxt "extruder_nr description"
-msgid "The extruder train used for printing. This is used in multi-extrusion."
-msgstr "Treno estrusore utilizzato per la stampa. Utilizzato nell’estrusione multipla."
+msgctxt "machine_extruder_start_pos_x label"
+msgid "Extruder Start Position X"
+msgstr "X posizione avvio estrusore"
 
-msgctxt "machine_nozzle_size description"
-msgid "The inner diameter of the nozzle. Change this setting when using a non-standard nozzle size."
-msgstr "Il diametro interno dell’ugello. Modificare questa impostazione quando si utilizza una dimensione ugello non standard."
+msgctxt "machine_extruder_start_pos_x description"
+msgid "The x-coordinate of the starting position when turning the extruder on."
+msgstr "La coordinata x della posizione di partenza all’accensione dell’estrusore."
 
-msgctxt "machine_nozzle_id description"
-msgid "The nozzle ID for an extruder train, such as \"AA 0.4\" and \"BB 0.8\"."
-msgstr "ID ugello per un treno estrusore, come \"AA 0.4\" e \"BB 0.8\"."
+msgctxt "machine_extruder_start_pos_y label"
+msgid "Extruder Start Position Y"
+msgstr "Y posizione avvio estrusore"
 
-msgctxt "machine_extruder_cooling_fan_number description"
-msgid "The number of the print cooling fan associated with this extruder. Only change this from the default value of 0 when you have a different print cooling fan for each extruder."
-msgstr "Il numero di ventole di raffreddamento stampa abbinate a questo estrusore. Modificarlo dal valore predefinito 0 solo quando si ha una ventola di raffreddamento diversa per ciascun estrusore."
+msgctxt "machine_settings label"
+msgid "Machine"
+msgstr "Macchina"
 
-msgctxt "machine_extruder_end_code_duration description"
-msgid "The time it takes to execute the end g-code, when switching away from this extruder."
-msgstr ""
+msgctxt "machine_settings description"
+msgid "Machine specific settings"
+msgstr "Impostazioni macchina specifiche"
```

Comment:
"Impostazioni specifiche macchina" sounds better to me (if 'specific' is related to 'settings' and not to 'machine'